.githooks/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I think we stopped using the githook in favor of the pre-commit.. The package should already be configure with it (https://github.com/lsst-ts/ts_externalscripts/blob/develop/.pre-commit-config.yaml) though, I must admit I am not really sure how they work.. maybe @r-owen can wave in here..

Reply:
I'm afraid that issue is still present. I wish I'd seen this before I merged my update. I'll make a small branch for this.

Reply:
Good catch. I fixed this (as an extra commit after merging my ticket).